diff --git a/node_modules/@types/react-native/index.d.ts b/node_modules/@types/react-native/index.d.ts
index 5e29f64..6485791 100755
--- a/node_modules/@types/react-native/index.d.ts
+++ b/node_modules/@types/react-native/index.d.ts
@@ -868,6 +868,8 @@ export interface LayoutRectangle {
     y: number;
     width: number;
     height: number;
+    left: number;
+    top: number;
 }
 
 // @see TextProps.onLayout
